### ISSUE TITLE
Compilation error for sha2.js

### ISSUE THREAD

**Issue by koto** [google/closure-library#402]
```
closure-library/closure/goog/crypt/sha2.js:201: ERROR - Bad type annotation. missing closing >
 *  @param {Array.<Uint32Array[80]>} precomputed_schedule An SHA2-32b schedule
```
It's gone if I just remove the "[80]".

**Comment by Ubehebe** [comment]
@koto Thanks for catching this. Here's what happened:

- Last week, I merged #391 by @coruus.
- The PR broke several unit tests (see PR discussion), and also doesn't compile, as you noticed.
- I rewrote the project's history using `git push --force` to make it so that the PR had never been merged.
- But I must not have done this correctly, since @coruus' original commit (e9dcd0ba641a8a3817e49b295a347954e47b3379) was from August, and it remained in the project's history.

I have now reverted e9dcd0ba641a8a3817e49b295a347954e47b3379 in 2a79f2bc6b43115631fc6a1000e027c9d863a24d and the library should build again. Please verify and closure this issue if it works.

**Comment by koto** [comment]
It works now, thanks!
